tests/Java.Interop.Tools.Maven-Tests/Extensions/TestDataExtensions.cs

```diff
@@ -0,0 +1,33 @@
+using System.Xml.Linq;
+using Java.Interop.Tools.Maven.Models;
+
+namespace Java.Interop.Tools.Maven_Tests.Extensions;
+
+static class TestDataExtensions
+{
+	public static Project CreateProject (Artifact artifact, Project? parent = null)
+	{
+		var p = new Project {
+			GroupId = artifact.GroupId,
+			ArtifactId = artifact.Id,
+			Version = artifact.Version,
+		};
+
+		if (parent is not null)
+			p.Parent = new Parent {
```

Comment:
This makes me wonder if there should be a `Parent(Parent)` constructor…

Reply:
Honestly, a better case could be made that `Project` should not have a `public` constructor, so I made it `internal` and changed this to generate xml that can be deserialized.